NN_chatCmd.js

```diff
@@ -131,12 +131,15 @@ nerthus.chatCmd.map["addGraf"] = function(ch)
 
 nerthus.chatCmd.extractUrlFromDecorator = function(text)
 {
-    if(text[0]=='<') //is text wrapped by html tag
-    {
-        var url = RegExp(/goToUrl\(\"(\S+)\"\)/).exec(text);
-        if(url) return "http://" + url[1];
+    let url = RegExp(/https\*Krzywi się\.\*(\S+)/).exec(text)
```

Comment:
to można wszystko jednym regexpem obrobić.
spróbuj
let url = RegExp(/(https?)\*Krzywi się\.\*(\S+)/).exec(text)
if(url) 
    return url[1] + "://" + url[2]